### ISSUE TITLE
Refer to parameters

### ISSUE THREAD

**Issue by NEFUtraffica** [koyeongmin/PINet_new#49]
Hello, author, I see you provide the training process, when I use Tusimple training, do I need to divide a part of the validation set additionally? I don't seem to see the validation set in your file structure, so how do I do the tuning, do I look at the test results during training to adjust the hyperparameters? Very much looking forward to your reply, I would appreciate it!
